ovirtlib4/defaults.py

```diff
@@ -1,2 +1,3 @@
 HOSTED_ENGINE_VM_NAME = "HostedEngine"
 ADMIN_USERNAME = "admin@internal"
+VM_START_TIMEOUT = 60  # Timeout wait for VM to start
```

Comment:
maybe 120 seconds to make sure we will not fail in case of slow VM

Reply:
I think 120 is too much if a faulty test will try to start some VMs and will wait 2 min for each try and even will have some retries, then this will increase testing time, I think it's good that the test will raise this issue, mostly now when we are trying to decrease testing time